core-plugins/docs/FTPDeprecated-batchsource.md

```diff
@@ -1,5 +1,6 @@
 # FTP Batch Source
 
+**NOTE**: This plugin is deprecated. Please use https://github.com/data-integrations/ftp-plugins instead.
```

Comment:
... Please use https://github.com/data-integrations/ftp-plugins instead, which is available as **FTP plugins** (> v3.0.0) from the Hub.

Reply:
fixed